DotNET/Complex Flow Examples/ocr-with-extract-text.cs

```diff
@@ -0,0 +1,62 @@
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+class Program
+{
+    private static readonly string apiKey = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"; // Your API key here
+
+    static async Task Main(string[] args)
+    {
+        using (var httpClient = new HttpClient { BaseAddress = new Uri("https://api.pdfrest.com") })
+        {
+            // Upload PDF for OCR
+            using var ocrRequest = new HttpRequestMessage(HttpMethod.Post, "pdf-with-ocr-text");
+
+            ocrRequest.Headers.TryAddWithoutValidation("Api-Key", apiKey);
+            ocrRequest.Headers.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
+            var ocrMultipartContent = new MultipartFormDataContent();
+
+            var pdfByteArray = File.ReadAllBytes("/path/to/file.pdf");
+            var pdfByteArrayContent = new ByteArrayContent(pdfByteArray);
+            ocrMultipartContent.Add(pdfByteArrayContent, "file", "file.pdf");
+            pdfByteArrayContent.Headers.TryAddWithoutValidation("Content-Type", "application/pdf");
+            ocrMultipartContent.Add(new StringContent("example_pdf-with-ocr-text_out"), "output");
+
+            ocrRequest.Content = ocrMultipartContent;
+            var ocrResponse = await httpClient.SendAsync(ocrRequest);
+
+            var ocrResult = await ocrResponse.Content.ReadAsStringAsync();
+            Console.WriteLine("OCR response received.");
+            Console.WriteLine(ocrResult);
+
+            dynamic ocrResponseData = JObject.Parse(ocrResult);
+            string ocrPDFID = ocrResponseData.outputId;
+
+            // Extract text from OCR'd PDF
+            using var extractTextRequest = new HttpRequestMessage(HttpMethod.Post, "extracted-text");
+
+            extractTextRequest.Headers.TryAddWithoutValidation("Api-Key", apiKey);
+            extractTextRequest.Headers.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
+            var extractTextMultipartContent = new MultipartFormDataContent();
+
+            extractTextMultipartContent.Add(new StringContent(ocrPDFID), "id");
```

Comment:
For whatever reason, I get an error back from the server due to lines such as this one, which use a `StringContent` object.

I tried a different solution taken from a different sample program -- it uses `ByteArrayContent` instead -- and it seems to be working:
```suggestion
            var byteArrayOption = new ByteArrayContent(Encoding.UTF8.GetBytes(ocrPDFID));
            extractTextMultipartContent.Add(byteArrayOption, "id");
```